packages/styles/src/components/GaugeCard/_gauge-card.scss

```diff
@@ -4,8 +4,10 @@
   display: flex;
   flex-wrap: wrap;
   font-size: carbon--type-scale(1);
-  height: 100%;
-  width: 100%;
+  flex-direction: row;
```

Comment:
@cal-smith It was my understanding that this package was auto generated from styles in the React package. Are we making edits directly to styles package now? Will the React and Angular styles now be divergent?

Reply:
or is this the newly generated styles that have been pulled from React package?

Reply:
I'm guessing it's pulled from styles changed in React, or it hasn't been updated after some changes ... fyi @Donisius

Reply:
Seems to be pulled over from the changes in react. I ran `sync` and `build` in the styles package, is there anything else that needs to be done (besides `postinstall` in angular) @cal-smith?

Reply:
Nope, that's it! This should be good then :+1: